devel/d3dmetal/Portfile

```diff
@@ -69,11 +69,13 @@ notes "
 
 platform darwin i386 {
     try {
```

Comment:
What is `try` doing? We don't use `try` anywhere else in MacPorts. We do use `catch` a lot…

Reply:
`try` by itself (i.e. with no handlers) does nothing, which is why the user in the ticket was seeing the error. `catch { blah }` is more or less equivalent to:
```Tcl
try {
    blah
} on error {} {
}
```